javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/TokenTextElement.java

```diff
@@ -21,57 +21,41 @@
 
 package com.github.javaparser.printer.lexicalpreservation;
 
-import com.github.javaparser.GeneratedJavaParserConstants;
 import com.github.javaparser.JavaToken;
-import com.github.javaparser.ast.Node;
 import com.github.javaparser.TokenTypes;
-
-import static com.github.javaparser.utils.Utils.EOL;
+import com.github.javaparser.ast.Node;
 
 class TokenTextElement extends TextElement {
-
-    private final int tokenKind;
-    private final String text;
-
-    public static TokenTextElement newLine() {
-        return new TokenTextElement(TokenTypes.eolTokenKind(), EOL);
-    }
+    private final JavaToken token;
```

Comment:
I'll directly wrap a JavaToken here, so the lex pres code gets a little more connected to the rest.